### ISSUE TITLE
Aria attributes not covered in current IDL

### ISSUE THREAD

**Issue by Garbee** [leobalter/cross-root-aria-delegation#10]
Looking over the spec, I see two attributes not covered that should be, `aria-expanded` and `aria-controls`. The use-case of these is making buttons that trigger expandable content like showing truncated entries or popups.

On a wider point however, do the attributes listed in https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes at least provide enough coverage to get started with for a full spec IDL? Or is there some better place to go that lists out all the current aria attributes?

**Comment by nolanlawson** [comment]
It does seem like the spec should cover the full list of `aria-*` attributes, including those that make ID references. The missing ones I see are:

- `aria-activedescendant`
- `aria-controls`
- `aria-describedby`
- `aria-details`
- `aria-errormessage`
- `aria-flowto`
- `aria-labelledby`
- `aria-describedby` (mentioned in part of the spec but missing in another part)
- `aria-owns`

(There may be others; I didn't do an exhaustive search.)

AIUI, there is no technical limitation to support `aria-*`; it's just a matter of capturing them all in the proposal.

As mentioned in https://github.com/leobalter/cross-root-aria-delegation/issues/13, we may also want to consider `for` and `role`.
